src/neml_interface.h

```diff
@@ -7,8 +7,7 @@
 #include "cp/singlecrystal.h"
 
 namespace neml {
-std::unique_ptr<NEMLModel> parse_xml_unique(std::string fname, std::string mname);
```

Comment:
We need to keep parse_xml_unique.  Should be:

std::unique_ptr<NEMLModel> parse_xml_unique(std::string fname, std::string mname);
std::unique_ptr<NEMLModel> parse_string_unique(std::string input, std::string mname);

i.e. keep them both.

Reply:
I checked and it will build with the amended commit

Reply:
Thanks, looks good.